WordPress/src/main/java/org/wordpress/android/ui/plans/PlansUtils.java

```diff
@@ -201,6 +201,7 @@ public static boolean downloadGlobalPlans() {
             @Override
             public void onResponse(JSONObject response) {
                 if (response != null) {
+                    // TODO: don't log response - only here for testing
```

Comment:
Is there a reason why we should not log the response? It doesn't contain any sensitive info in it.
It would be helpful to have it in the log, for all those cases where users will contact us saying the "Plan" menu item is not visible on mobile.